### DISCUSSION TITLE
两台电脑的备份数据可以合并吗？

### DISCUSSION THREAD

**Discussion by Qwerty4401** [ronggang/PT-Plugin-Plus#1398] [Q&A] [answered]
工作日用电脑A更新各站数据，非工作日用电脑B更新各站数据。如果用电脑A备份以后，在电脑B恢复，数据是合并还是覆盖呢？

**Comment by ted423** [comment]
我感觉不行，因为刷新时间不同可能产生不同的数据

**Reply by Qwerty4401** [reply]
> 我感觉不行，因为刷新时间不同可能产生不同的数据

要是能合并就好了，多设备数据可以打通。

**Reply by fqh1995** [reply]
以一个设备为主，建议在常用设备，手机或者pc为主

**Comment by IITII** [comment, marked as answer]
目前没有数据合并功能, 数据恢复是覆盖.
> 只需要保证同一台电脑每日刷新, 上传备份即可.
